content/articles/quarkus-configuration-profiles/index.md

```diff
@@ -0,0 +1,576 @@
+  - [Creating a new Quarkus project](#creating-a-new-quarkus-project)
+    - [Project structure](#project-structure)
+  - [Set up some configurations](#set-up-some-configurations)
+  - [Testing the configurations](#testing-the-configurations)
+- [Configuring the application using Quarkus configuration profiles](#configuring-the-application-using-quarkus-configuration-profiles)
+  - [Quarkus' Configuration Profile formats](#quarkus-configuration-profile-formats)
+  - [Using default configuration profiles](#using-default-configuration-profiles)
+  - [Using user defined configuration profile](#using-user-defined-configuration-profile)
+- [Setting up and using variables in different profiles](#setting-up-and-using-variables-in-different-profiles)
+  - [Setting up variables](#setting-up-variables)
+  - [Injecting the variables](#injecting-the-variables)
+  - [Testing the variables](#testing-the-variables)
+    - [Testing in the dev profile](#testing-in-the-dev-profile)
+    - [Testing in the prototype profile](#testing-in-the-prototype-profile)
+- [Configuration profiles in YAML files](#configuration-profiles-in-yaml-files)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Key takeaways
+Some points to be taken in by the end of the article follow up include:
+- What is Quarkus Configuration Profiles
+- Importance of configuration profiles
+- Some ways of setting up configuration profiles in Quarkus. These include setting the profiles in the `application.properties` and 'YAML' files.
+- Easily navigating between the profiles set during the development stages.
+
+### Prerequisites
+For one to follow up with the code examples and explanations given in the article, the following will be required:
+- Java Development Kit installed on the machine. The recommendation is the latest JDK in the market. JDK 17 is used for the article.
+- A good Java IDE. The recommendation is the latest version of the IntelliJ Ultimate edition. This choice is because it supports MicroProfile, among which Quarkus is one of them. IntelliJ version `2021.2.2` was used for the article, as seen in the screenshot captured.
+- Familiar with databases.
+- A good internet connection. The internet is for fetching indexes for quick development. It will also be used in the generation of the project.
+
+> **NOTE**:- These pre-requisites may change as time goes by ever since the day that the article is published.
+> Make sure to follow up on _the latest versions_ of the technologies mentioned above.
+> The appearance of the applications used may also differ based on the release dates.
+
+### What is Quarkus Configuration Profiles?
+
+Quarkus, like any other MicroProfile, follows the MicroProfile specifications. This property makes it to be able to allow developers to set the variables found in the whole project seamlessly so that they can be accessed in the project scope during different project life cycles.
```

Comment:
This sentence is quite hard to read, could consider adding a transition?

Reply:
Added